recipe/meta.yaml

```diff
@@ -1,109 +1,115 @@
 {% set name = "imagecodecs" %}
-{% set version = "2021.8.26" %}
+{% set version = "2023.1.23" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: e87994d918ea951a589ec46648c24d9c7e56b7b5c2466d93f3b07883f0ecea19
+  sha256: 1b4591839a2f5c90467e50ebe54117cbd9be42c18864abdf85fc7743223ac5b2
   patches:
     - 0001-remove-blosc2-codec.patch
     - 0002-patch-aarch64-s390x-codecs.patch   # [s390x or aarch64]
+    # setup.py links jpeg8 against 'jpeg' which is the name for the static build (jpeg.lib)
+    # instead it should link against 'libjpeg' which is shared (libjpeg.dll)
+    - 0003-jpeg8_use_shared_libjpeg.patch     # [win]
 
 build:
-  # these are no longer supported.
-  skip: true   # [py<37 or win32]
-  number: 2
+  skip: true   # [py<38]
+  number: 0
   entry_points:
     - imagecodecs=imagecodecs.__main__:main
+  ignore_run_exports:
+    # Windows build needs brotli but does not use this one
+    - libbrotlicommon   # [win or s390x]
+    # jxrlib for win is only built as static
+    - jxrlib            # [win]
 
 requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
     - pkg-config
-    - m2w64-toolchain  # [win]
-    - patch            # [unix]
-    - m2-patch         # [win]
+    - patch       # [not win]
+    - m2-patch    # [win]
   host:
-    - setuptools >=18.0
-    - m2w64-pkg-config  # [win]
+    - setuptools
     - python
     - pip
-    - numpy-devel {{ numpy }}  # [not (osx and arm64) and py<=39]
-    - numpy {{ numpy }}        # [(osx and arm64) or py>39]
-    - cython
-    - zlib
-    - bzip2
-    - xz
-    - zstd
-    - lz4-c                 # [not win]
-    - lz4-c >=1.9.3         # [win]
-    - libpng
-    - libwebp
-    - libwebp-base
-    - jxrlib
-    - jpeg
-    - giflib
-    - openjpeg
-    - blosc
-    - lcms2  # [not win32]
-    - libaec
-    - brotli
-    - libzopfli
-    - charls          # [not s390x]
-    - snappy
+    - wheel
+    - numpy {{ numpy }}
+    - cython 0.29.35
+    - zlib {{ zlib }}
+    - bzip2 {{ bzip2 }}
+    - xz {{ xz }}
+    - zstd 1.5.5
+    - lz4-c 1.9.4
+    - libpng {{ libpng }}
+    - libwebp {{ libwebp }}
+    - libwebp-base 1.2.4
+    - jxrlib 1.1
+    - jpeg {{ jpeg }}
+    - giflib {{ giflib }}
+    - openjpeg {{ openjpeg }}
+    - blosc 1.21.3
+    - lcms2 2.12
+    - libaec 1.0.4
+    - libbrotlidec 1.0.9      # [not (win or s390x)]
+    - libbrotlienc 1.0.9      # [not (win or s390x)]
+    - brotli 1.0.9            # [win or s390x]
```

Comment:
I'm kind of curious as to how it could work on Linux and mac... If the headers are in `brotli`...

Reply:
@JeanChristopheMorinPerso I wonder the same. Searching through the upstream repo did not help much.

I can see that `brunsli` is set for `[not win and not s390x]`, and both require then `brotli`.

Searching quickly through `brunsli`, I can see that it does bring in `brotli`:
https://github.com/search?q=repo%3Agoogle%2Fbrunsli%20brotli&type=code

Reply:
Then we need it as a directly dependency to get the headers. And we'll probably want it as a runtime dep too.

Reply:
Ok, I will restore `brotli` for all, and restore the runtime dep

Reply:
Done, it is building